acceptance/lib/helper.rb

```diff
@@ -154,13 +154,17 @@ def puppet_server_collect_data(host, relative_path)
       if version.to_i >= 15
         use_journalctl = true
       end
+    when /^sles$/
+      if version.to_i >= 12
+        use_journalctl = true
+      end
     end
 
     destination = File.join("./log/latest/puppetserver/", relative_path)
     FileUtils.mkdir_p(destination)
     scp_from master, "/var/log/puppetlabs/puppetserver/puppetserver.log", destination
     if use_journalctl
-      puppetserver_daemon_log = on(master, "journalctl -u puppetserver").stdout.strip
+      puppetserver_daemon_log = on(master, "journalctl -u puppetserver",  :acceptable_exit_codes => [0,1]).stdout.strip
```

Comment:
(nitpick) Two spaces in `,  :acceptable`; not important, but if you fixup this PR at all maybe grab it?